vunit/sim_if/xcelium.py

```diff
@@ -0,0 +1,419 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Copyright (c) 2014-2021, Lars Asplund lars.anders.asplund@gmail.com
+
+"""
+Interface for the Cadence Xcelium simulator
+"""
+
+from pathlib import Path
+from os.path import relpath
+import os
+import subprocess
+import logging
+from ..exceptions import CompileError
+from ..ostools import write_file, file_exists
+from ..vhdl_standard import VHDL
+from . import SimulatorInterface, run_command, ListOfStringOption
+from .cds_file import CDSFile
+
+LOGGER = logging.getLogger(__name__)
+
+
+class XceliumInterface(  # pylint: disable=too-many-instance-attributes
+    SimulatorInterface
+):
+    """
+    Interface for the Cadence Xcelium simulator
+    """
+
+    name = "xcelium"
+    supports_gui_flag = True
+    package_users_depend_on_bodies = False
+
+    compile_options = [
+        ListOfStringOption("xcelium.xrun_vhdl_flags"),
+        ListOfStringOption("xcelium.xrun_verilog_flags"),
+    ]
+
+    sim_options = [ListOfStringOption("xcelium.xrun_sim_flags")]
+
+    @staticmethod
+    def add_arguments(parser):
```

Comment:
@cmarqu taking into account that Xcelium is the newest, and Incisive could be deprecated sometime in the future, I defined the CLI arguments for both here. I put a note into incisive.py. What do you think?